deps.windows/10-mbedtls.ps1.disabled

```diff
@@ -0,0 +1,90 @@
+param(
+    [string] $Name = 'mbedtls',
+    [System.Version] $Version = '2.24.0',
+    [string] $Uri = 'https://github.com/ARMmbed/mbedtls.git',
+    [string] $Hash = '523f0554b6cdc7ace5d360885c3f5bbcc73ec0e8',
+    [array] $Patches = @(
+        @{
+            PatchFile = "${PSScriptRoot}/patches/mbedtls/0001-enable-alt-threading-mode.patch"
+            HashSum = "dd811e3e406430d7140d04b6c23eee6dc1a7ec8bca70d3a6b0d9db55cc9f4fdc"
+        }
+        @{
+            PatchFile = "${PSScriptRoot}/patches/mbedtls/0002-add-alt-threading-header-file.patch"
+            HashSum = "1c42a3bd74ada543f8852ccf8e55f482022f08d3888e0b5d0d101ab0beff409d"
+        }
+    ),
+    [switch] $Shared = $false
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Clean {
+    Set-Location $Path
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Patch {
+    Log-Information "Patch (${Target})"
+
+    Set-Location $Path
+
+    $Patches | ForEach-Object {
+        $Params = $_
+        Safe-Patch @Params
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+    $OnOff = @('OFF', 'ON')
+    $Options = @(
+        $CmakeOptions
+        "-DUSE_SHARED_MBEDTLS_LIBRARY=$($OnOff[$Shared.isPresent])"
```

Comment:
Yeah see above, I'll do a pass over the Powershell scripts and remove the `isPresent` calls.

Reply:
Since this is actually a Switch Parameter for this script, removing `.IsPresent` here should work. However, I do notice that this is different from the other scripts where you do not use a Switch Parameter, and instead use a $script scope variable. So perhaps you meant this to be $script scope and not have a Switch Parameter present for this script file?